package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -289,7 +290,7 @@ dhcp_host_add() {
 
 	config_get leasetime "$cfg" leasetime
 
-	xappend "--dhcp-host=$macs${networkid:+,net:$networkid}${broadcast:+,set:needs-broadcast}${tag:+,set:$tag}${ip:+,$ip${hostid:+,[::$hostid]}}${name:+,$name}${leasetime:+,$leasetime}"
+	xappend "--dhcp-host=$macs${duid:+,id:$duid}${networkid:+,net:$networkid}${broadcast:+,set:needs-broadcast}${tag:+,set:$tag}${ip:+,$ip${hostid:+,[::$hostid]}}${name:+,$name}${leasetime:+,$leasetime}"
```

Comment:
Because 'duid' is already a well known parameter and documented. I didn't make this up, the name comes from https://wiki.openwrt.org/doc/uci/dhcp. Where 'duid' is specified, it is specific for DHCPv6 and should not be used for DHCPv4. So I repeat, if we want to use this, there needs to be a different uci parameter since for DHCPv4 and DHCPv6 this value has different meanings.